### PR TITLE
TaskList converted 

### PR DESCRIPTION
**Pull request by Faustina-Acheampong**
I have converted the TaskList and updated tasks.tsx  but they may need reviewing just to be sure.

### CODE REVIEW

**Comment by emmy-codes** [comment]
As long as the checks pass from Git, it's safe to merge :D but also good practice to have someone else review!

File: src/App.tsx
Comment (gmoraleslondono) [inline src/App.tsx:1]: I would suggest removing the changes on this file since it is not related to the convert TaskList to TS.
It is adding semicolons (;)

File: src/redux/tasks.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { createSlice } from '@reduxjs/toolkit'
 import type {PayloadAction} from '@reduxjs/toolkit'
 
-interface Task {
+export interface Task {
```
Comment (gmoraleslondono) [inline src/redux/tasks.tsx:4]: Usually if you need to use the same types in different files/ components it is a good practice to create a shared types file. Where you can create all the types and then export them from there
